src/components/Skill/SmallSkill.js

```diff
@@ -0,0 +1,12 @@
+import React, { Component } from "react";
+
+export default class SmallSkill extends Component {
```

Comment:
Maybe its an overkill to have a component for this compared to just keep it with in the view or is it because it is going to be used some were else?